app/views/notices/shop_employee_notices/16b_initial_employee_open_enrollment_begins.html.erb

```diff
@@ -53,16 +53,16 @@
 <%= render partial: 'notices/shared/paragraph_with_headers', :locals =>
   { contents: [""], heading: 'For Questions or Assistance:', sub_heading: ''} %>
 
+<%= render partial: 'notices/shared/paragraph', :locals => { content: "Please contact your broker for further assistance.You can also contact The #{Settings.site.short_name} with any questions:"} %>
+<%= render partial: 'notices/shared/unordered_list', :locals => { listings:
+                                                                   ["By calling :#{Settings.contact_center.ma_phone_number}.", "TTY :#{Settings.contact_center.tty} ", "Online at : <a href='#{Settings.site.ma_website}'>#{Settings.site.ma_website}</a>"] } %>
```

Comment:
you should use a helper method like we've been doing in app/helpers/config rather than use Settings.contact_center.ma_phone_number